web/templates/settings/profile_field_choice.hbs

```diff
@@ -1,5 +1,5 @@
 <div class='choice-row movable-row' data-value="{{value}}">
-    <span class="move-handle">
+    <span class="move-handle {{#if new_empty_choice_row}} hide_icon {{/if}}">
```

Comment:
New CSS classes should all use dashes, and be globally unique in the codebase. But we don't need a new one; the existing `invisible` class does the job here.